.drone.yml

```diff
@@ -482,6 +484,7 @@ steps:
       from_secret: docker_user
   depends_on:
   - copy-packages-for-docker
+  - upload-cdn-assets-ubuntu
```

Comment:
There's no step called upload-cdn-assets-ubuntu, a bug?

Reply:
hm.. this task maybe has a different sfx variable? the sfx enterprise2 etc stuff was very messy :)